services/nginx/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM nginx:1.15.0-alpine
```

Comment:
Do we need nginx as well?

Reply:
nginx and references to it are now removed